src/bedriftsmeny/Bedriftsmeny.tsx

```diff
@@ -27,22 +27,32 @@ const Bedriftsmeny: FunctionComponent<EgneProps> = (props) => {
             return juridiskEnhetMedUnderEnheterArray;
         };
         if (props.organisasjoner && props.organisasjoner.length > 0) {
-            byggTre(props.organisasjoner.sort((a, b) => a.Name.localeCompare(b.Name))).then((juridiskEnhetMedUnderEnheterArray) => {
-                const organisasjonstre = juridiskEnhetMedUnderEnheterArray;
-                if (organisasjonstre.length > 0) {
-                    setOrganisasjonstre(organisasjonstre);
+            byggTre(props.organisasjoner.sort((a, b) => a.Name.localeCompare(b.Name))).then(
+                (juridiskEnhetMedUnderEnheterArray) => {
+                    const organisasjonstre = juridiskEnhetMedUnderEnheterArray;
+                    if (organisasjonstre.length > 0) {
+                        setOrganisasjonstre(organisasjonstre);
+                    }
                 }
-            });
+            );
         }
     }, [props.organisasjoner]);
 
-    const visVirksomhetsvelger = organisasjonstre && organisasjonstre.length > 0
-        && props.organisasjoner && props.organisasjoner?.length>0;
+    const visVirksomhetsvelger =
+        organisasjonstre &&
+        organisasjonstre.length > 0 &&
+        props.organisasjoner &&
+        props.organisasjoner?.length > 0;
 
     return (
-        <div className="bedriftsmeny">
+        <div className="bedriftsmeny" role="banner">
             <div className="bedriftsmeny__inner">
-                <Innholdstittel className="bedriftsmeny__tittel">{sidetittel}</Innholdstittel>
+                <Innholdstittel
+                    className="bedriftsmeny__tittel"
+                    role="heading"
+                    aria-label={sidetittel}>
+                    {sidetittel}
+                </Innholdstittel>
```

Comment:
role og aria-label på h1 er unødvendig :) h1 = heading

Reply:
role kan være unødvendig, men aria-label mener jeg trengs her også

Reply:
Hvorfor det? :) Jeg tenker at det er helt unødvendig, for skjermleseren leser jo opp teksten uten at aria-label er der

Reply:
Forandret til aria-label="Innholdstittel for applikasjon". Slik at en skjermleser i tillegg vil lese opp konteksten til heading.